### PR TITLE
Added general Error class + fixed & improved log function

### PR DESCRIPTION
**Pull request by TheGreyDiamond**
Added general Error class and fixed & improved log functions

### CODE REVIEW

File: OSTools.py [hunk 1]
```diff
@@ -159,5 +164,12 @@ def warn(msg):
...
\ No newline at end of file
+    
+    def info(self, msg):
+        ''' Makes a log output, without showing anything on the EV3 screen'''
+        global logMsg
+        sound(SoundFile.GENERAL_ALERT)
+        if True:
+            timeStamp = strftime("%d.%m.%Y %H:%M:%S")
+            print("[" + timeStamp + " Info]", exception, msg)
+
```
Comment (XilefTech) [inline OSTools.py:174]: please put all the sound allerts into the if statemnts for the screen printing...

File: movements.py [hunk 2]
```diff
@@ -10,7 +10,7 @@ def execute(params, *args):
 
     if (OSTools.getBatteryVoltage() <= 7500):
         log.warn("Please charge the battery. Only " + str(OSTools.getBatteryVoltage(human=True)) + " V left. You need at least 7.5 Volts.")
-        return "Battery too low"
+        return RobotErrors.Battery.tooLow
```
Comment (XilefTech) [inline movements.py]: robotError.RobotErrors.[...]

File: robotError.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (XilefTech) [inline robotError.py:1]: why into another file? u can use OSTools I guess.... 
More files = more uploading time

File: README.md [hunk 4]
```diff
@@ -25,3 +25,6 @@ Just fill in your robot Parameters in the config.py file and everything else lik
 - steer the robot with a remote control of some kind (actually not sure what kind), save the steered path and let it repeat it
 - use a FLL mode with separate program slots according to the Tasks on the Field and an extra mode for executing them in a specific order at the competition itself
 
+## Contributors
+- Xilef
```
Comment (XilefTech) [inline README.md:29]: *XilefTech

File: robotError.py (hunk 3 above)
Comment (TheGreyDiamond) [reply robotError.py:1]: Makes the code more accessible to others and more clear.

File: movements.py (hunk 2 above)
Comment (TheGreyDiamond) [reply movements.py]: Outdated please update

File: OSTools.py (hunk 1 above)
Comment (TheGreyDiamond) [reply OSTools.py:174]: Sorry, I don't get what you mean?

File: README.md (hunk 4 above)
Comment (TheGreyDiamond) [reply README.md]: Done

File: OSTools.py (hunk 1 above)
Comment (XilefTech) [reply OSTools.py:174]: the sound(SoundFile.GENERAL_ALERT) into the if statement in which the messages are printed on the screen

Comment (TheGreyDiamond) [reply OSTools.py:174]: Gonna close this now, if you don't react in a timely manner.

**Pull request merged**